sessions/css-flexbox/navigation/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./css/styles.css" />
+    <title>Static Template</title>
+  </head>
+  <body>
+    <header class="header">
+      <h1>My Personal Website</h1>
+      <nav>
+        <ul class = 'navigation__menu'>
+          <li class="navigation__link">
+            <a  href="#about">About</a>
+          </li>
+          <li class="navigation__link">
+            <a  href="#skills">Skills</a>
+          </li>
+          <li class="navigation__link">
+            <a  href="#biography">Biography</a>
+          </li>
+          <li class="navigation__link">
+            <a  href="#contact">Contact</a>
+          </li>
+        <ul>
```

Comment:
It is a bit weird that if we only give the `ul` a class and not the `nav` to start with a BEM element and not a block. I would probably give the `ul` the class: `navigation` instead of `navigation__menu`.

Reply:
As discussed in the other PR, me and Thomas thought that we sould show them best practice from the start. But I definetly see your point.

Reply:
I see the point as well, but I would agree with Felix and Thomas.

Additionally, if we explain the structure beforehand, the students should be able to work with it here as well.